src/platform/i386/timer.c

```diff
@@ -131,6 +145,9 @@ timer_calibration(void)
 		timer_cycles_per_hpetcyc = (TIMER_ERROR_BOUND_FACTOR * cycles_per_tick) / hpetcyc_per_tick;
 		printk("Timer calibrated:\n\tCPU cycles per HPET tick: %ld\n\tHPET ticks in %d us: %ld\n",
 		       timer_cycles_per_hpetcyc/TIMER_ERROR_BOUND_FACTOR, TIMER_DEFAULT_US_INTERARRIVAL, hpetcyc_per_tick);
+
+		timer_disable(TIMER_PERIODIC);
+		timer_disable(TIMER_PERIODIC);
```

Comment:
Yes, I understand. I looked up online to figure that out sometime back and found one in linux kernel archive, who said the same thing, "HPETs are a complete disaster..." like here: http://lxr.free-electrons.com/source/arch/x86/kernel/hpet.c#L375

Will fix it, Thank you.